oss_src/unity/lib/unity_sarray.cpp

```diff
@@ -1622,6 +1622,16 @@ std::shared_ptr<unity_sarray_base> unity_sarray::sample(float percent,
   return logical_filter(seq);
 }
 
+
+std::shared_ptr<unity_sarray_base> unity_sarray::hash(int random_seed) {
```

Comment:
Was going to. But we have "int" *everywhere* going quite deep. sample takes an int as well.
Fixing that will be a much larger code change than I want to do now.